projects/optic/src/utils/git-utils.ts

```diff
@@ -6,7 +6,7 @@ export const hasGit = async (): Promise<boolean> =>
       if (err || stderr || !stdout) resolve(false);
       resolve(true);
     };
-    const command = `git -v`;
+    const command = `git --version`;
```

Comment:
This wasn't working for me for some reason - not sure if there's version differences but `git --version` runs correctly, before this wasn't picking up the correct config